### PR TITLE
#167453183 Refactor codebase

### PR DESCRIPTION
**Pull request by tolulope-od**
#### What does this PR do?
Refactors codebase to use ES6+ syntax
Adds babel to transpile ES6 code
Adds run server script

#### Description of Task to be completed?
Refactor the codebase to use ES6+ syntax and add babel to transpile code

#### How should this be manually tested?
Assuming you have cloned the repo and you are in the directory on your terminal, do the following: 
```bash
# fetch this branch
$ git fetch origin ch-refactor-codebase-167453183

# install dependencies
$ npm install

# run script to start the dev server
$ npm run server
```
After following these steps, you should see console output on your terminal with the server running on port 3000

#### Any background context you want to provide?
Due to the fact that the MongoDB server has not been configured, there would also be an error on the console about unhandled promise rejections. That's expected due to the above-stated reason

#### What are the relevant pivotal tracker stories?
[#167453183](https://www.pivotaltracker.com/story/show/167453183)

#### Screenshots (if appropriate)
*Server entry point before refactoring to ES6*

Screenshot 2019-07-23 at 10 43 20

*After Refactoring to ES6*

Screenshot 2019-07-23 at 11 56 20

#### Questions:
N/a

### CODE REVIEW

File: config/passport.js [hunk 1]
```diff
@@ -1,26 +1,29 @@
+import passport from "passport";
 const passport = require("passport");
```
Comment (PelumiAlesh) [inline config/passport.js]: You forgot to delete this line

Comment (tolulope-od) [reply config/passport.js]: Thanks. I'll do that and push again

**Review by OdunayoOkebunmi — COMMENTED** [review]
Looks good to me.

**Review by mekzy-o — APPROVED** [review]
Looks good to me.

**Review by OdunayoOkebunmi — APPROVED** [review]

File: routes/index.js [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+import { Router } from "express";
+import apiRoutes from "./api";
 
-router.use("/api", require("./api"));
+const router = Router();
 
-module.exports = router;
+router.use("/api", apiRoutes);
+
+export default router;
```
[the remark this answers is not in the document]
Comment (ocranbillions) [reply routes/index.js:8]: why is hound flagging the exports?

**Comment by mazma1** [comment]
@tolulope-od  Why am I seeing Hound comments? Has it been integrated already?

**Review by PelumiAlesh — APPROVED** [review]

**Comment by tolulope-od** [comment]
> Why am I seeing Hound comments? Has it been integrated already?

It was just added as an app to the repo, but the rules haven't been configured yet

**Comment by mazma1** [comment]
@tolulope-od I'll close this PR because we should be working on the `develop` branch like I said in the morning. I'll switch the default branch back to `develop`, raise your PR again and I'll merge.

**Pull request closed without merging**